go/vt/vtctl/grpcvtctldserver/server.go

```diff
@@ -1153,6 +1153,28 @@ func (s *VtctldServer) GetKeyspace(ctx context.Context, req *vtctldatapb.GetKeys
 	}, nil
 }
 
+// GetFullStatus is part of the vtctlservicepb.VtctldServer interface.
```

Comment:
nit: keep the method list sorted, please 🙇

Reply:
Fixed.